practices/pairProgramming/readme.md

```diff
@@ -0,0 +1,33 @@
+# Pair Programming
+
+Pair programming is the concept of two team members working on a single story/feature. Often, this is two developers working back and forth (there are numerous patterns/strategies to use) to deliver business value. Some high value pairs could be frontend and backend specialists. Each can help drive their specialty areas as well as help train the other. Additionally, QA and Devs can pair to ensure that their features are delivered with appropriate tests in place.
```

Comment:
pretty sure the commit this is from is in another PR here, if any changes were made since the PR the order of the merges might matter wrt merge conflicts

Reply:
Ok! Yeah the changes to the front page file shouldn't create conflicts, but I'm not sure.